libcloudforensics/aws.py

```diff
@@ -1065,6 +1065,71 @@ def ShareWithAWSAccount(self, aws_account_id):
     )
 
 
+class AWSCloudTrail:
+  """Class representing an AWS CloudTrail service.
+
+  Attributes:
+    aws_account (AWSAccount): The AWS account to use.
+  """
+
+  def __init__(self, aws_account):
+    """Initialize an AWS CloudTrail client.
+
+    Args:
+      aws_account (AWSAccount): The AWS account to use.
+    """
+
+    self.aws_account = aws_account
+
+  def LookupEvents(self, qfilter=(),
+                   starttime=datetime(1800, 1, 1),
+                   endtime=datetime.now()):
+    """Lookup events in the CloudTrail logs of this account.
+
+    Example usage:
+      # pylint: disable=line-too-long
+      # qfilter = [{'AttributeKey':'Username', 'AttributeValue':'root'}]
+      # starttime = datetime(2020,5,5,17,33,00)
+      # LookupEvents(qfilter=qfilter, starttime=starttime)
+      # Check documentation for qfilter details
+      # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html#CloudTrail.Client.lookup_events
+
+    Args:
+      qfilter (tuple): Optional. Filter for the query including 1 AttributeKey and AttributeValue.
+      starttime (datetime): Optional. Start datetime to add to query filter
+      endtime (datetime): Optional. End datetime to add to query filter
+
+    Returns:
+      list(dict): A list of events
+    """
+
+    next_token = ''
+    events = []
+
+    client = self.aws_account.ClientApi('cloudtrail')
```

Comment:
Move `'cloudtrail'` to its own constant at the top of the file along with the EC2/KMS constants that are defined